packages/core-modules/contracts/modules/ElectionModule.sol

```diff
@@ -61,20 +64,28 @@ contract ElectionModule is IElectionModule, ElectionSchedule, OwnableMixin {
         nominees.remove(msg.sender);
     }
 
-    function isNominated(address candidate) external view override returns (bool) {
-        return _getCurrentEpoch().nominees.contains(candidate);
-    }
+    // ---------------------------------------
+    // Vote functions
+    // ---------------------------------------
 
-    function getNominees() external view override returns (address[] memory) {
-        return _getCurrentEpoch().nominees.values();
-    }
+    function elect(address[] calldata candidates) external override onlyInPeriod(ElectionPeriod.Vote) {
+        _validateCandidates(candidates);
 
-    /* solhint-disable */
-    function elect(address[] memory candidates) external override onlyInPeriod(ElectionPeriod.Vote) {
-        // TODO
+        uint votePower = _getVotePower(msg.sender);
+        if (votePower == 0) {
+            revert NoVotePower();
+        }
```

Comment:
Maybe this check can be put before the call to `_validateCandidates` because potentially costs less gas, and in case of error it would be cheaper if thrown first.

Reply:
Implemented